src/binding.cc

```diff
@@ -1600,13 +1601,13 @@ void v8_inspector__StringBuffer__DELETE(v8_inspector::StringBuffer* self) {
   delete self;
 }
 
-const v8_inspector::StringView* v8_inspector__StringBuffer__string(
-    v8_inspector::StringBuffer* self) {
-  return &self->string();
+three_pointers_t v8_inspector__StringBuffer__string(
+    const v8_inspector::StringBuffer& self) {
+  return make_pod<three_pointers_t>(self.string());
 }
```

Comment:
Related to v8 upgrade?

Reply:
https://github.com/v8/v8/commit/844fe8f7d965fcb305bd1b71fe6b34628fc4d1fd